pkgs/development/python-modules/faker/default.nix

```diff
@@ -1,59 +1,45 @@
-{ lib, buildPythonPackage, fetchPypi, pythonOlder,
-# Build inputs
-dateutil, six, text-unidecode, ipaddress ? null
-# Test inputs
-, email_validator
+{ lib
+, buildPythonPackage
+, fetchPypi
+, dateutil
+, text-unidecode
 , freezegun
-, mock
-, more-itertools
 , pytestCheckHook
-, pytestrunner
 , ukpostcodeparser
 , validators
 }:
 
-assert pythonOlder "3.3" -> ipaddress != null;
-
 buildPythonPackage rec {
   pname = "Faker";
-  version = "4.18.0";
+  version = "5.5.1";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0raxw6mgvf9523v7917zqw76vqnpp0d6v3i310qnjnhpxmm78yb2";
+    sha256 = "1bskhmiir1ajipj7j535j2mxgnp6s3mxbvlag4aryj9zbhgg1c19";
   };
 
-  nativeBuildInputs = [ pytestrunner ];
+  propagatedBuildInputs = [
+    dateutil
+    text-unidecode
+  ];
+
   checkInputs = [
-    email_validator
     freezegun
     pytestCheckHook
     ukpostcodeparser
     validators
-  ]
-  ++ lib.optionals (pythonOlder "3.3") [ mock ]
-  ++ lib.optionals (pythonOlder "3.0") [ more-itertools ];
+  ];
 
   # avoid tests which import random2, an abandoned library
   pytestFlagsArray = [
     "--ignore=tests/providers/test_ssn.py"
```

Comment:
Did you check if this still applies?

Reply:
Just checked. Still an issue.

Reply:
Unfortunately, [yes](https://github.com/joke2k/faker/blob/master/tests/providers/test_ssn.py#L10).